kamon-akka-http-server/src/main/scala/kamon/akka/http/server/AkkaHttpServerMetrics.scala

```diff
@@ -0,0 +1,33 @@
+/* ===================================================
+ * Copyright © 2013 the kamon project <http://kamon.io/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ========================================================== */
+package kamon.akka.http.server
+
+import kamon.http.HttpServerMetrics
+import kamon.metric.EntityRecorderFactory
+import kamon.metric.instrument.InstrumentFactory
+
+class AkkaHttpServerMetrics(instrumentFactory: InstrumentFactory) extends HttpServerMetrics(instrumentFactory) {
+  def recordConnection(bindingAddress: String): Unit =
+    counter(bindingAddress).increment()
+
+  def openConnections = minMaxCounter("open-connections")
```

Comment:
Yeap, it probably can be just `openRequests`. I like the idea of having this metric! Also, this can be a val, to avoid looking it up in the entity map every time it is needed.

One more thing is that we (Diego and I) thought to do some changes to the HttpServerMetrics, not just for Spray but for Play and akka-http as well. Our main concern at the moment is that the current implementation is just based on conventions about how to name certain metrics and you end up with counters named like [this](https://github.com/kamon-io/Kamon/blob/master/kamon-spray/src/test/scala/kamon/spray/SprayServerMetricsSpec.scala#L68), which seem rather unfortunate. After we introduced support for tags, it feels like the correct way to represent HTTP server metrics would be a group of smaller entities with well defined properties and identifying tags rather than the current single, bloated, convention based entity. Also, by doing this we could add a tag for the binding port, meaning that we could get separate metrics for different bindings made by the same application, which we don't do with the current Spray and Play modules.